tests/integration/test_network_plugin.py

```diff
@@ -40,15 +40,30 @@ def setUp(self):
     def tearDown(self):
         super(NatRulesOperationsTestCase, self).tearDown()
 
-    def test_nat_rules_create_delete(self):
-        self.assertNotIn(self.public_ip, collectExternalIps(
-            self._get_gateway()))
+    def test_nat_rules_create_delete_with_explicit_ip(self):
+        self.ctx.node.properties['floatingip'].clear()
+        self.ctx.node.properties['floatingip'].update({'public_ip': self.public_ip,
+                                                       'gateway': 'M966854774-4471'})
```

Comment:
Please use vcloud_integration_test_config.json (https://github.com/cloudify-cosmo/tosca-vcloud-plugin/blob/net-develop/examples/vcloud_integration_test_config.json.example) to pass things like public ip and gateway to the test.